test/OpenTelemetry.Exporter.Geneva.Tests/GenevaMetricExporterOptionsTests.cs

```diff
@@ -69,5 +69,27 @@ public void InvalidPrepopulatedDimensions()
             expectedErrorMessage = $"Value provided for the dimension: DimensionKey exceeds the maximum allowed limit of {GenevaMetricExporter.MaxDimensionValueSize} characters for dimension value.";
             Assert.Equal(expectedErrorMessage, invalidDimensionValueException.Message);
         }
+
+        [Fact]
+        public void MetricExportIntervalValidationTest()
+        {
+            Assert.Throws<ArgumentOutOfRangeException>(() =>
+            {
+                var exporterOptions = new GenevaMetricExporterOptions
+                {
+                    MetricExportIntervalMilliseconds = 500,
```

Comment:
Minor - consider testing on the boundary:

```suggestion
                    MetricExportIntervalMilliseconds = 999,
```

Reply:
@Yun-Ting I have committed this suggestion.